.editorconfig

```diff
@@ -6,7 +6,7 @@
 charset = utf-8
 end_of_line = lf
 indent_size = 4
-indent_style = tab
+indent_style = space
```

Comment:
We should be using psr2 now; I think some upgrades were missing from cms module though. I should run a formatter over it.